.travis.yml

```diff
@@ -2,10 +2,10 @@ language: node_js
 sudo: false
 install: true
 jdk: oraclejdk11
-node_js: "10.16.0"
+node_js: "10.15.3"
```

Comment:
@wouter-admiraal-sonarsource 10.16.0 (current LTS) is not available on Travis ?

Reply:
We align it on sonar-enterprise, which uses 10.15.3.

Reply:
Shouldn't we align the other way and have the latest LTS ?

Reply:
We could, although we don't care that much (IMHO). It's only for building, not running any application itself. And I realize we're speaking of the `.travis.yml` file, which yes, we can def. set to the LTS 😅 . But we can keep 10.15.3 in `package.json`.